src/Symfony/Component/ErrorHandler/Resources/views/exception_full.html.php

```diff
@@ -0,0 +1,43 @@
+<!-- <?= $_message = sprintf('%s (%d %s)', $exceptionMessage, $statusCode, $statusText); ?> -->
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="<?= $this->charset; ?>" />
```

Comment:
XML-style closing tags should be replaced by HTML-like ones if the XHTML profile isn't used.

Reply:
The codebase is using this style everywhere so I'm going to keep it as is for now.